### PR TITLE
move logging to CONSOLE_BRIDGE macros

### PR DESCRIPTION
**Pull request by pholthau**
### Description

I replaced all logging (error, inform, warn, debug) with the following command. No code was altered otherwise:
```
find -type f -print0 | xargs -0 sed -i 's/logError/CONSOLE_BRIDGE_logError/g'
```

This should fix compilation on gentto, see ros/ros-overlay#509

Please explain the changes you made, including a reference to the related issue if applicable

### Checklist
- [ ] **Required**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)

I did not modify any code otherwise, so I did not re-check the format.

### CODE REVIEW

**Comment by rhaschke** [comment]
Some of the old-style logging commands were already replaced before, such that you have `CONSOLE_BRIDGE_CONSOLE_BRIDGE_log*` now. Hence, you need to apply a more advanced sed command.

**Comment by pholthau** [comment]
Thanks, Robert! Looked good when rushing trough the output. Sorry

**Comment by rhaschke** [comment]
Please fix clang-format issues:
http://moveit.ros.org/documentation/contributing/code

File: moveit_core/constraint_samplers/src/constraint_sampler_manager.cpp [hunk 1]
```diff
@@ -116,9 +118,10 @@ constraint_samplers::ConstraintSamplerPtr constraint_samplers::ConstraintSampler
       JointConstraintSamplerPtr sampler(new JointConstraintSampler(scene, jmg->getName()));
       if (sampler->configure(jc))
       {
-        logDebug("Temporary sampler satisfying joint constraints for group '%s' allocated. Looking for different types "
-                 "of constraints before returning though.",
-                 jmg->getName().c_str());
+        CONSOLE_BRIDGE_logDebug("Temporary sampler satisfying joint constraints for group '%s' allocated. Looking for "
+                                "different types "
```
Comment (rhaschke) [inline moveit_core/constraint_samplers/src/constraint_sampler_manager.cpp]: clang formatting cannot nicely handle line breaks in literals.
@pholthau May I ask you to manually adjust weird line breaks like these?

**Comment by pholthau** [comment]
Hope that I found all of them!

**Comment by pholthau** [comment]
Thanks for advising me, Robert! Also, I didn't know that you also do line breaks semantically after sentences and/or linking words. So thanks for cleaning that as well.

**Comment by v4hn** [comment]
Is it just me who thinks that these function names are ridiculous?
I would have very much preferred central definitions that forward to the appropriate console_bridge API...

**Comment by rhaschke** [comment]
This _is_ the new, official console_bridge API. I'm not sure if I really should have introduced MoveIt-specific macro definitions for logging instead.

**Comment by davetcoleman** [comment]
Yes, I think the names are silly. IMHO we should just use ROS_DEBUG_* and ROS_INFO_* even in ``moveit_core``. Reasons are 1) for uniformity throughout code base and 2) console_bridge is broken and doesn't allow you to properly filter out different levels of namespaces like the ROS_ equivalent does.

**Comment by rhaschke** [comment]
> IMHO we should just use `ROS_DEBUG_*` and `ROS_INFO_*` even in `moveit_core`.

I disagree. moveit_core was designed to be independent of ROS and thus should not bring in a ROS dependency via `ros_console`. If `console_bridge` is broken, it should be fixed.

**Comment by v4hn** [comment]
> This is the new, official console_bridge API

Well, it's the official shortcut macros at least, I know.
That's why I phrased my comment the way I did...

> moveit_core was designed to be independent of ROS and thus should not bring in a ROS dependency via ros_console. If console_bridge is broken, it should be fixed.

Yes. I only complained that every 10th line or so in `moveit_core` now tells the developer in full length that our logging backend is `CONSOLE_BRIDGE`. I agree that we shouldn't switch to the ROS backend here.
I was only looking for further opinions.

**Comment by davetcoleman** [comment]
Getting off topic but...

> moveit_core was designed to be independent of ROS

Sure, but the reality is that it requires catkin, roslib, rostime, and a host of ROS-generated ros messages. On top of that, a large part of moveit is not in moveit-core so moveit is barely useful without ROS as it currently is written. So in my current opinion it seems silly we pretend to keep the two separate. For the most part, it just makes the code base needlessly complex by requiring an extra layer of interfaces and differing programming styles that increases the difficulty for developers to contribute.

**Comment by rhaschke** [comment]
Valid point. Do you suggest to switch to `ros_console` then?
As far as I remember, `ros_console` relays to `console_bridge` as long as `ros::init()` wasn't called yet. Can anybody confirm?

**Pull request merged**